test/src/test/crypto.h

```diff
@@ -16,7 +16,7 @@ static bool validate_aesgcm()
 	char *decrypted = calloc(1, strlen(plain) + 1);
 	char *tag1 = calloc(1, 16);
 
-	MTY_AESGCM *aes1 = MTY_AESGCMCreate(password1);
+	MTY_AESGCM *aes1 = MTY_AESGCMCreate(password1, 16);
```

Comment:
Should you add tests for `keySize = 32` and for when `keySize != 16 && keySize != 32`?